.github/workflows/e2e.yml

```diff
@@ -69,3 +69,66 @@ jobs:
       - run: cd ../ci-${{ matrix.trpc }}-${{ matrix.tailwind }}-${{ matrix.nextAuth }}-${{ matrix.prisma }} && pnpm build
         env:
           NEXTAUTH_SECRET: foo
+
+  build-t3-app-with-bun:
+    runs-on: ubuntu-latest
+
+    name: "Build and Start T3 App with Bun"
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+
+      # First install everything and build the CLI with Node
+      - name: Install Node.js
+        uses: actions/setup-node@v3
+        with:
+          node-version: 18
+
+      - uses: pnpm/action-setup@v2.2.4
+        name: Install pnpm
+        id: pnpm-install
+        with:
+          run_install: false
+
+      - name: Get pnpm store directory
+        id: pnpm-cache
+        run: |
+          echo "pnpm_cache_dir=$(pnpm store path)" >> $GITHUB_OUTPUT
+
+      - uses: actions/cache@v3
+        name: Setup pnpm cache
+        with:
+          path: ${{ steps.pnpm-cache.outputs.pnpm_cache_dir }}
+          key: ${{ runner.os }}-pnpm-store-${{ hashFiles('**/pnpm-lock.yaml') }}
+          restore-keys: |
+            ${{ runner.os }}-pnpm-store-
+
+      - name: Install dependencies
+        run: pnpm install
+
+      - run: pnpm turbo --filter=create-t3-app build
+
+      # Then, run the CLI and build the generated app with Bun
+      # Let's just build a full app with Bun, we don't need the matrix here
+      - uses: oven-sh/setup-bun@v1
+        with:
+          bun-version: latest
+
+      - run: cd cli && bun run dist/index.js ../../ci-bun --default
+
+      - name: We should have a Bun lockfile
+        run: |
+          if [ ! -f "../../ci-bun/bun.lockb" ]; then
+            echo "Bun lockfile not found"
+            exit 1
+          fi
```

Comment:
@Zeko369 I added this to make sure we get a bun lockfile which should indicate the CLI uses bun to install the dependencies, so that when we then run `bun run build` we should have all the dependencies and such installed using Bun and the build script should succeed